wpeverest/user-registration
# UR-1476 Enhance - All forms list table design

### All Submissions:






### Changes proposed in this Pull Request:
Currently, we are using the default WordPress list table design which seems to be old-fashioned compared to our updated   Form Builder and Settings Design. This PR adds some design changes to the All Forms listing page and table that makes it sleek and stylish.

### How to test the changes in this Pull Request:

1. Visit the User Registration -> All Forms page and verify if there are no design issues.
2. Check to see if the functionality of the list table is intact.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - All forms list table design
